Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -45,13 +45,15 @@ If this value is empty, each pod will get an ephemeral directory to store their
     * `port`: Allows to change the default port where the dashboard is served
     * `ssl`: Whether to serve the dashboard via SSL, ignored on Ceph versions older than `13.2.2`
 * `monitoring`: Settings for monitoring Ceph using Prometheus. To enable monitoring on your cluster see the [monitoring guide](../../Storage-Configuration/Monitoring/ceph-monitoring.md#prometheus-alerts).
-    * `enabled`: Whether to enable prometheus based monitoring for this cluster
+    * `enabled`: Whether to enable prometheus based monitoring for external or internal cluster
     * `externalMgrEndpoints`: external cluster manager endpoints
     * `externalMgrPrometheusPort`: external prometheus manager module port. See [external cluster configuration](#external-cluster) for more details.
     * `rulesNamespace`: Namespace to deploy prometheusRule. If empty, namespace of the cluster will be used.
       Recommended:
         * If you have a single Rook cluster, set the `rulesNamespace` to the same namespace as the cluster or keep it empty.
         * If you have multiple Rook clusters in the same Kubernetes cluster, choose the same namespace to set `rulesNamespace` for all the clusters (ideally, namespace with prometheus deployed). Otherwise, you will get duplicate alerts with duplicate alert definitions.
+    * `port`: internal prometheus manager module port. See [server_port](#https://docs.ceph.com/en/latest/mgr/prometheus/#confval-mgr-prometheus-server_port).
```

Comment:
Why would the port need to be changed from the default? If using host networking? And we don't need the ceph doc link since it doesn't provide more details. 
```suggestion
    * `port`: The internal prometheus manager module port where the prometheus mgr module listens. The port may need to be configured when host networking is enabled. 
```